components/event-service/server/server.go

```diff
@@ -15,6 +15,7 @@ import (
 
 var (
 	defaultRegistry = event.Registry{
+		config.EventFeedEventName:      []string{config.EventFeedEventName},
```

Comment:
Instead of adding a line here for each type of chef-server event and directing it to the EventFeedEventName type, the chef-server events are going to use the EventFeedEventName type directly.